### PR TITLE
[Artwork] Wrap tabs in Theme

### PR DESCRIPTION
**Pull request by damassi**
Follow-up to https://github.com/artsy/force/pull/2886

Wraps the new tabs component in a theme so that styling is correct:

screen shot 2018-09-14 at 11 48 31 am

### CODE REVIEW

File: Dangerfile [hunk 1]
```diff
@@ -9,6 +9,11 @@ unless github.branch_for_base == 'release'
   warn("Please assign someone to your PR", sticky: false) unless has_assignee
 end
 
+# Warn if adding new files that aren't .ts or .tsx
+if git.added_files.include? ".js"
+  warn('All new files should be `.ts` or `.tsx`. JavaScript has been depreciated.')
+end
```
Comment (damassi) [inline Dangerfile]: @orta - this _seems_ right -- any clue why danger didn't warn due to added file at the bottom?

File: src/desktop/components/react/stitch_components/ReactionArtworkDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from "react"
+import { Theme } from "@artsy/palette"
+import { ContextProvider } from "reaction/Artsy"
+import { ArtworkDetailsQueryRenderer as Renderer } from "reaction/Apps/Artwork/Components/ArtworkDetails"
+import { data as sd } from "sharify"
+
+export const ReactionArtworkDetails = props => {
+  return (
+    <Theme>
```
Comment (damassi) [inline src/desktop/components/react/stitch_components/ReactionArtworkDetails.tsx]: @oxaudo - For style stuff, can wrap component in a `<Theme>` from `@artsy/palette`

File: Dangerfile (hunk 1 above)
Comment (orta) [reply Dangerfile]: Think it needs to be a glob

Comment (ashfurrow) [reply Dangerfile]: Ah, the `added_files` is an array, so you're asking if a file named `.js` has been added. You'll have to do a `select` or something on the array to check if it includes any file that ends in `.js`.

Comment (orta) [reply Dangerfile]: we ended up adding a danger js dangerfile + peril to this repo instead, as Danger ruby was removed when moving to hokusai

File: src/desktop/components/react/stitch_components/ReactionArtworkDetails.tsx (hunk 2 above)
Comment (oxaudo) [reply src/desktop/components/react/stitch_components/ReactionArtworkDetails.tsx]: Oh... my bad with `.js` file and thanks for the `<Theme> addition! Seems pretty straightforward.

**Pull request merged**